src/cmds/cloud/createAppToken.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console */
+import yargs from 'yargs';
+import chalk from 'chalk';
+import { Authenticator } from '@cesarbr/knot-cloud-sdk-js';
+
+const createAppToken = async (args) => {
```

Comment:
Wouldn't it be better just update the existing `createToken` operation to support creating 'app' tokens?

Reply:
Done!